1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,30 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+  let displayEl = document.getElementById("content");
```

Comment:
The code looks really good, except for the varible names (and the listner, see below) - I would name them more descriptively for instance selectEl could become colourSelectDropdown or something like that, then you can easily scan the code